aio/aio-proxy/aio_proxy/search/dirigeant.py

```diff
@@ -0,0 +1,80 @@
+from elasticsearch_dsl import query
+
+
+def search_dirigeant(search, **kwargs):
+    dirigeants_filters = []
+    boost_queries = []
+
+    for key, value in kwargs.items():
+        if key == "nom_dirigeant" and value is not None:
```

Comment:
if value is None: 
   continue

switch key:
    case: "nom_dirigeant"
       do ...
    case ...

Reply:
I used dict.get() instead.